eclair-node/src/main/scala/fr/acinq/eclair/gui/controllers/SendPaymentController.scala

```diff
@@ -68,7 +75,7 @@ class SendPaymentController(val handlers: Handlers, val stage: Stage, val setup:
       Try(amount.toLong) match {
         case Success(amountLong) =>
           if (GUIValidators.validate(paymentRequestError, "Amount must be greater than 0", amountLong > 0)
-            && GUIValidators.validate(paymentRequestError, "Amount must be less than 4 294 967 295 msat (~0.042 BTC)", amountLong < 4294967295L)) {
+            && GUIValidators.validate(paymentRequestError, "Amount must be less than 4 294 967 295 msat (~0.042 BTC)", amountLong < PaymentRequest.maxAmountMsat)) {
```

Comment:
"Amount must be less than 4 294 967 295 msat (~0.042 BTC)" -> "Amount must be less than 4 294 967 295 msat (~0.042 BTC)"